corporate/views/session.py

```diff
@@ -56,3 +57,28 @@ def start_card_update_stripe_session_for_realm_upgrade(
             "stripe_session_id": stripe_session.id,
         },
     )
+
+
+@authenticated_remote_realm_management_endpoint
+@typed_endpoint
+def start_card_update_stripe_session_for_remote_realm_upgrade(
+    request: HttpRequest,
+    billing_session: RemoteRealmBillingSession,
+    *,
+    manual_license_management: Json[bool] = False,
+) -> HttpResponse:  # nocoverage
+    metadata = {
+        "type": "card_update",
+        # Not passed for remote realms.
+        # "user_id": user.id,
+    }
+    stripe_session = billing_session.create_stripe_update_card_for_realm_upgrade_session(
+        metadata, Session.CARD_UPDATE_FROM_UPGRADE_PAGE, manual_license_management
+    )
+    return json_success(
+        request,
+        data={
+            "stripe_session_url": stripe_session.url,
+            "stripe_session_id": stripe_session.id,
+        },
```

Comment:
We can probably reduce duplication further by making `create_stripe_update_card_for_realm_upgrade_session` return the `json_success` bit rather than the `stripe_session`, maybe with renaming it. I suppose the metadata could also be a billing session function.

Reply:
Ok, will add it to next PR.